tests/test_mcp_compliance.py

```
@@ -0,0 +1,284 @@
+#!/usr/bin/env python3
+"""MCP Compliance Tests - Ensure all components use real MCP, no mocks"""
+
+import pytest
+import asyncio
```

Ruff (0.11.9)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)